fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -229,6 +234,55 @@ public void setIsAllowNull(boolean isAllowNull) {
         this.isAllowNull = isAllowNull;
     }
 
+    public String getMetaDefaultValue(List<String> extras) {
+        String defaultValue = FeConstants.null_string;
+        if (this.getDefaultValue() != null) {
+            defaultValue = this.getDefaultExpr();
+        } else if (this.getDefaultExpr() != null) {
+            if ("now()".equalsIgnoreCase(this.getDefaultExpr())) {
+                defaultValue = "CURRENT_TIMESTAMP";
+                extras.add("DEFAULT_GENERATED");
+            }
+        }
+        return defaultValue;
+    }
+
+    public String getDefaultExpr() {
+        return defaultExpr;
+    }
+
+    public void setDefaultExpr(String defaultExpr) {
+        this.defaultExpr = defaultExpr;
+    }
```

Comment:
This function should not exposed to users.
I think the only interface about default value should be getDefaultValue, no matter that it is a default value or default expr.